meta-balena-common/recipes-bsp/u-boot/patches/0001-common-Disable-u-boot-console.patch

```diff
@@ -0,0 +1,27 @@
+From c4f8fa2011b2b9878aaa1f263be228a6e2bfe9d3 Mon Sep 17 00:00:00 2001
+From: Florin Sarbu <florin@balena.io>
+Date: Tue, 19 May 2020 10:43:01 +0200
+Subject: [PATCH] common: Disable u-boot console
+
+Upstream-Status: Inappropriate [configuration]
+Signed-off-by: Florin Sarbu <florin@balena.io>
+---
+ common/init/board_init.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/common/init/board_init.c b/common/init/board_init.c
+index 526fee3..d11942a 100644
+--- a/common/init/board_init.c
++++ b/common/init/board_init.c
+@@ -120,6 +120,8 @@ void board_init_f_init_reserve(ulong base)
+ 	 * Use gd as it is now properly set for all architectures.
+ 	 */
+ 
++	gd->flags |= (GD_FLG_SILENT | GD_FLG_DISABLE_CONSOLE);
```

Comment:
Thinking about the desire to merge development and production images, would it make sense to apply this conditionally on an environment variable?
Something like it's done at https://github.com/u-boot/u-boot/blob/ed9a3aa6452f57af65eb74f73bd2a54c3a2f4b03/common/console.c#L756

Reply:
@alexgg sure we can guard it like that. Shall we use maybe `CONFIG_DISABLE_CONSOLE` since it's more to what we do here?

Reply:
hi @floion I was thinking on a future with a single image for development and production images, where we have a dashboard switch to silence the console. For this to work we would need to have a way to change the behaviour at runtime, so something like a u-boot environment setting came to mind.
It's not something to hold the merge though.